hack/lib/etcd.sh

```diff
@@ -161,19 +161,21 @@ kube::etcd::install() {
     fi
 
     if [[ ${os} == "darwin" ]]; then
-      download_file="etcd-v${ETCD_VERSION}-darwin-amd64.zip"
+      download_file="etcd-v${ETCD_VERSION}-${os}-${arch}.zip"
```

Comment:
Do we wanna force amd64 binaries on Darwin for now?

Reply:
(Not sure how quickly etcd will get Darwin arm builds)

Reply:
@endocrimes no, i'd like to leave this as-is.